pkg/config/containers.conf

```diff
@@ -799,3 +799,11 @@ default_sysctls = [
 # TOML does not provide a way to end a table other than a further table being
 # defined, so every key hereafter will be part of [machine] and not the
 # main config.
+
+[farms]
```

Comment:
the section needs to be added to the feebsd file as well.

Reply:
Fixed